stdlib/SparseArrays/src/sparsematrix.jl

```diff
@@ -1383,23 +1341,20 @@ function _sparse_findprevnz(m::SparseMatrixCSC, i::Integer)
 end
 
 
-function _sprand(r::AbstractRNG, m::Integer, n::Integer, density::AbstractFloat, rfn)
-    m, n = Int(m), Int(n)
-    (m < 0 || n < 0) && throw(ArgumentError("invalid Array dimensions"))
-    0 <= density <= 1 || throw(ArgumentError("$density not in [0,1]"))
-    j, colm  = 1, 0
-    rowval = randsubseq(r, 1:(m*n), density)
-    nnz = length(rowval)
-    colptr = Vector{Int}(undef, n + 1)
+function sparse_increasing!(I::Vector{Ti}, V::Vector, m::Int, n::Int) where Ti
```

Comment:
Maybe `sparse_sortedlinearindices!` would be more descriptive, since that is what `I` is here?